hw3/tests/test_hw3_t03.py

```diff
@@ -0,0 +1,118 @@
+from hw3.hw3_t03 import make_filter
+
+
+def test_positive_for_1_subject():
+    sample_data = [
+        {
+            "name": "Bill",
+            "last_name": "Gilbert",
+            "occupation": "was here",
+            "type": "person",
+        },
+        {"is_dead": True, "kind": "parrot", "type": "bird", "name": "polly"},
+    ]
+    assert make_filter({"name": "polly", "type": "bird"}).apply(sample_data) == [
+        {"is_dead": True, "kind": "parrot", "type": "bird", "name": "polly"}
+    ]
+
+
+def test_both_characters_by_diff_key_value_pairs():
```

Comment:
Don't try to shorten words. Especially such commonly used in [IT](https://git-scm.com/docs/git-diff).